app/views/content/teaching-internship-providers/_listing.html.erb

```diff
@@ -17,6 +17,8 @@
 
 <p> If you’re interested in teaching a different subject, you can still <a href="/train-to-be-a-teacher/get-school-experience">get school experience</a> and find out what it’s like to be in a classroom.</p>
 
+<p> “Towards the end of the internship, I was able to teach parts of lessons that I had planned, and I really started to find my feet at the front of the classroom!” - Ben, former teaching intern. </p>
+
 <h2> What you'll cover</h2>
 
 <p>You'll get a feel for school life, which could include:</p>
```

Comment:
this is a bit repetitive - says `which could include` twice in quick succession?